api/db/changelog/db.changelog-179-drop-billing-project-buffer-entry-table.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+<changeSet author="calbach" id="changelog-179-drop-billing-project-buffer-entry-table">
+    <dropTable tableName="billing_project_buffer_entry"/>
```

Comment:
I have had to do a couple of bulk cleanups in lower environments where this table has been very useful - e.g. when we accidentally created a bunch of spurious projects that we thought were failures. I'm slightly worried that we may have a need for it later, if we discover some kind of regression / bug. Thoughts on leaving this around for a bit?

Reply:
that's good point. I also realized we may want to delete those available projects, that also needs this table